core/src/test/java/org/elasticsearch/search/suggest/AbstractSuggestionBuilderTestCase.java

```diff
@@ -211,6 +213,22 @@ public void testFromXContent() throws IOException {
         }
     }
 
+    protected Tuple<MapperService, SB> mockMapperServiceAndSuggestionBuilder(
```

Comment:
I took another stab at simplifying [this](https://github.com/areek/elasticsearch/commit/3940c898422eb4d2f031cf05dc16f1bdd6d7978f), Thoughts?